app/models/article.rb

```diff
@@ -902,8 +901,7 @@ def bust_cache(destroying: false)
   end
 
   def calculate_base_scores
-    self.hotness_score = 1000 if hotness_score.blank?
-    self.spaminess_rating = 0 if new_record?
+    self.hotness_score = 1000 if hotness_score.blank
```

Comment:
```suggestion
    self.hotness_score = 1000 if hotness_score.blank?
```

Reply:
Hmm the "Commit suggestion" button is disabled for some reason, but I'll add this in myself 👍